### ISSUE TITLE
Fix test specifications in build scripts

### ISSUE THREAD

**Issue by elfring** [coccinelle/coccinelle#116]
[The build approach was updated to some degree](https://github.com/coccinelle/coccinelle/commit/3277ba7a1d03dc53de6d3b676d915b498fbe97b2 "Next Makefile") for the Coccinelle software. This change introduced some bugs. I would appreciate another fix for the following siutation.

[A configuration script](https://github.com/coccinelle/coccinelle/blob/436cc65a41ce2c5885cc7f1ee09b6c14fac1981c/configure.ac#L518 "Build specification") can provide a specific information so far.

```
…
Then, to test coccinelle, run for example:
…
Or when coccinelle is not installed yet:
  $ … ./scripts/spatch -sp_file demos/…
…
```

It seems that the mentioned executable file is not available any more in this directory.
:crystal_ball: How would you like to adjust [affected places](https://github.com/coccinelle/coccinelle/blob/99447494981473615f95236b9b9a01abf7aeb017/scripts/spatch.sh.in "Update candidate?")?

**Comment by JuliaLawall** [comment]
On Fri, 13 Oct 2017, Markus Elfring wrote:

>
> The build approach was updated to some degree for the Coccinelle software.
> This change introduced some bugs. I would appreciate another fix for the
> following siutation.
>
> A configuration script can provide a specific information so far.
>
> …
> Then, to test coccinelle, run for example:
> …
> Or when coccinelle is not installed yet:
>   $ … ./scripts/spatch -sp_file demos/…
> …
>
> It seems that the mentioned executable file is not available any more in
> this directory.

Why do you consider this to be a bug?  To my knowledge there was never any
promise that scripts/spatch would exist.

julia

**Comment by elfring** [comment]
>To my knowledge there was never any promise that scripts/spatch would exist.

How does this response fit to information in the file “configure.ac”?

**Comment by JuliaLawall** [comment]
On Fri, 13 Oct 2017, Markus Elfring wrote:

>
>       To my knowledge there was never any promise that scripts/spatch
>       would exist.
>
> How does this response fit to information in the file “configure.ac”?

I know nothing about this file.  To my understanding, scripts/spatch was a
means to an end.  If the end is achieved in some other way, there there is
no problem.  configure.ac is an implementation, not a specificaton of what
is desired.

julia

**Comment by elfring** [comment]
>I know nothing about this file.

How can this be if you are the maintainer of this software for years?


>configure.ac is an implementation, not a specificaton of what is desired.

Did you discuss any corresponding software updates already in your research team?

**Comment by JuliaLawall** [comment]
On Fri, 13 Oct 2017, Markus Elfring wrote:

>
>       I know nothing about this file.
>
> How can this be if you are the maintainer of this software for years?
>
>       configure.ac is an implementation, not a specificaton of what is
>       desired.
>
> Did you discuss any corresponding software updates already in your research
> team?

I try to avoid any thing related to the makefile.

julia

**Comment by vgambier** [comment]
Fixed in 07e224f35
